src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -273,6 +273,14 @@ public int getMaxBadMessages() {
         return getInt("secor.consumer.max_bad_messages", 1000);
     }
 
+    public boolean getBadMessagesTopicEnabled() {
+        return getBoolean("secor.consumer.bad_message.topic.enabled", false);
+    }
+
+    public String getBadMessagesTopicSuffix() {
+        return getString("secor.consumer.bad_message.topic.suffix", "_BAD_MESSAGES");
```

Comment:
Instead of defaulting, we should throw an error if this property is not specified.